modules/standard/LinkedLists.chpl

```diff
@@ -203,6 +203,52 @@ record LinkedList {
      return ret;
    }
 
+  /*
+    find `e` in the list.
+
+    :arg e: The element to be checked for
+    :return: True if the item was found, otherwise returns false
+    :rtyle: `bool`
+   */
+  proc contains(e : eltType): bool {
+    for item in this {
+      if (e == item) {
+        return true;
+      }
+    }
+    return false;
+  }
+  
+  /*
+    Returns the data stored in the first element of the list
+    
+    :return: the data stored at the front of the list
+    :rtype: `ref eltType`
+    :throws SystemError: if the list is empty
+   */
+  proc ref front() ref: eltType throws {
```

Comment:
You should rename this method to `first` instead of `front` so that it matches `list.first`.